rpcs3/rpcs3qt/kernel_explorer.cpp

```diff
@@ -889,3 +898,42 @@ void kernel_explorer::Update()
 	final_touches(root);
 	root->setExpanded(true);
 }
+
+void kernel_explorer::log(u32 level, QTreeWidgetItem* item)
+{
+	if (!item)
+	{
+		// 
+		m_log_buf = windowTitle();
+		m_log_buf[m_log_buf.indexOf('|')] = ':'; // Replace the first '|' with ':'
```

Comment:
Why not just use : to begin with?

Reply:
It doesn't make sense anyway since we don't wanna log the translated title.
Just recreate this again with a regular string and getidandtitle

Reply:
Done.